volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -0,0 +1,331 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import logging
+
+from struct import unpack
```

Comment:
Please import struct, for the same reasons as below for the exceptions module.  We try only to import modules so that people can easily see what's part of this module's namespace, what isn't and if it isn't what namespace it does belong to...